homeassistant/components/climate/melissa.py

```diff
@@ -123,6 +124,14 @@ def state(self):
             return self.melissa_state_to_hass(
                 self._cur_settings[self._api.STATE])
 
+    @property
+    def device_state_attributes(self):
+        """Add the attributes that's available."""
+        data = dict()
+        if self._data:
+            data['humidity'] = self._data['humidity']
```

Comment:
This should be returned from the `current_humidity` property. It's an official supported value.

Reply:
The platform does not support to target humidity just have the sensor value, and unfortunately it will not get passed if it does not support the feature.

Reply:
I meant that the api that melissa provides do not support setting humidity.
But if the value exist we should be able to set it without supporting target_humidity yes.

Reply:
Sorry, yes, that's what I meant

Reply:
So new SUPPORT_ or should it be set if it exist?

Reply:
No new `SUPPORT_`, those are only for services. Just add it to attributes if it's not `None`